usage_examples/tests/conftest.py

```diff
@@ -0,0 +1,5 @@
+from spacy_llm.tests.conftest import (
+    pytest_addoption,  # noqa: F401
```

Comment:
I usually add something like:

```toml
[tool.ruff.per-file-ignores]
# Ignore unused imports in __init__ files
"__init__.py" = ["F401"]
```

to my `pyproject.toml` file, that way I don't have to repeat it every time.

Reply:
That makes sense! Didn't know this is possible. In general I'd like to have all unused imports flagged though, so I won't add this for now. If this is a frequently occurring situation, I'll update the `ruff` config accordingly.